plugins/qeta-backend/src/database/DatabaseQetaStore.ts

```diff
@@ -2812,6 +2828,7 @@ export class DatabaseQetaStore implements QetaStore {
       url: val.url ?? undefined,
       images: additionalInfo[5]?.map(r => r.id),
       experts: additionalInfo[6],
+      published: val.published ? (val.published as Date) : undefined,
```

Comment:
It returns the published date now, but it should show it also in the UI instead `created` if it's set.